website/i18n/ko/docusaurus-plugin-content-docs/current/providers/provider.mdx

```diff
@@ -12,90 +12,70 @@ import unoptimizedPreviousButton from "!!raw-loader!/i18n/ko/docusaurus-plugin-c
 import optimizedPreviousButton from "!!raw-loader!/i18n/ko/docusaurus-plugin-content-docs/current/providers/provider/optimized_previous_button.dart";
```

Comment:
Anything inside the `providers` folder is bound to be deleted (and other things flagged as "old" in the sidebar)

So while I appreciate the thought, I'm not sure that's worth working on